examples/conf_example.c

```diff
@@ -23,6 +23,17 @@
 
 #include <openssl/engine.h>
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000
+int main()
+{
+    fprintf(stderr, "This wolfEngine example is not compatible with versions "
```

Comment:
As discussed on Slack, let's see if it works to call `OPENSSL_config()` for older versions of OpenSSL.  It would be nice to have this as a working example for 1.0.2 users as well.

Reply:
You were right....it worked!!  Pushed a change to use that.